bin.src/write_sne_truth.py

```diff
@@ -3,15 +3,62 @@
 Script to write the SNe truth catalog.
 """
 import argparse
+import os
+import sys
 import desc.sims_truthcatalog
+from datetime import datetime as dt
 
 parser = argparse.ArgumentParser(description="Write a truth catalog (sqlite) for SNe given a SNe parameters db file")
 parser.add_argument('outfile', type=str,
                     help='Filename for output sqlite file')
 parser.add_argument('sne_db_file', type=str,
                     help='Filename for input SNe parameters db file')
+parser.add_argument('--dry-run', action='store_true',
+                    help='no computation or database write')
+parser.add_argument('--variable-table', action='store_true',
+                    help='create and fill variable truth table')
+parser.add_argument('--aux-table', action='store_true',
+                    help='create and fill auxilliary truth table')
+parser.add_argument('--max-parallel', default=1, dest='max_parallel', type=int,
+                    help='max # of processes to be run in parallel; applies only to variability table (default: %(default)s)')
+parser.add_argument('--row-limit', default=None, type=int,
+                    help='write at most this number per process (applies only to variability table (default: no limit)')
+parser.add_argument('--no-summary', action='store_true', help='speed up debugging other tables')
+parser.add_argument('--verbose', action='store_true',
+                    help='write debug output while making aux table')
 
 args = parser.parse_args()
 
-writer = desc.sims_truthcatalog.SNeTruthWriter(args.outfile, args.sne_db_file)
-writer.write()
+print('write_sne_truth invoked with arguments')
+for e in dir(args):
```

Comment:
Since code similar to this block is used in the other scripts in `bin.src`, it might be useful to put it in a function that can be imported.

Reply:
Agreed.  I had meant to do that and forgot.